map/elevation_info.cpp

```diff
@@ -54,7 +54,7 @@ ElevationInfo::ElevationInfo(Track const & track)
   FillProperty(properties, kLowestPointKey, m_minAltitude);
   FillProperty(properties, kHighestPointKey, m_maxAltitude);
 
-  uint8_t difficulty;
+  uint8_t difficulty = UINT8_MAX;
```

Comment:
Instead of introducing unexpected values, would it be better to either return true from the FillProperty if it succeeds, or initialize the difficulty to Difficulty::Unknown by default?

Reply:
> or initialize the difficulty to Difficulty::Unknown by default?

The relevant `m_difficulty` is already inited to `Difficulty::Unknown`, the temporary simply shouldn't be used, if `FillProperty` can't do its job for whatever reason.
Is that OK, if I specialized `FillProperty` to handle `Difficulty` properly?